bentoml/cli/bento.py

```diff
@@ -47,7 +47,7 @@ def _print_bento_table(bentos, wide=False):
             for artifact in bento.bento_service_metadata.artifacts
         ]
         apis = [
-            f'{api.name}<{api.handler_type}>'
+            f'{api.name}<{api.input_type}:{api.output_type}>'
```

Comment:
We may also need to update the Web UI for this, @yubozhao  could you help identify the files that need similar change?

Reply:
Yep.
We would need to update the table view here: https://github.com/bentoml/BentoML/blob/8d49f11e02ae89100569e954ff27cd697e0a6bd0/bentoml/yatai/web/client/src/components/BentoServiceTable.tsx#L11

and the API table:
https://github.com/bentoml/BentoML/blob/8d49f11e02ae89100569e954ff27cd697e0a6bd0/bentoml/yatai/web/client/src/components/BentoServiceDetail/ApisTable.tsx#L57